tools/build-tools/src/bumpVersion/utils.ts

```diff
@@ -251,3 +252,8 @@ export async function runPolicyCheckWithFix(gitRepo: GitRepo){
         fatal(`Policy check needed to make modifications. Please create PR for the changes and merge before retrying.\n${afterPolicyCheckStatus}`);
     }
 }
+
+export function prereleaseSatisfies(packageVersion: string, range: string) {
+    // Pretend that the current package is latest prerelease (zzz) and see if the version still satisfies.
+    return semver.satisfies(`${packageVersion}-zzz`, range)
```

Comment:
Does this still satisfy if the zzz are 3 different characters?

Reply:
The goal here is to specify a `prerelease` version that is will specify all any ranges of prerelease.
Since semver specifies that alphabetical versions being "larger" then the numbers, `zzz` will cover most cases.
Of course we can a prerelease range that this won't work (e.g. ^0.blah.0-zzzz).  But since we don't use alphabetical prerelease version (and even if we do , we probably won't use anything larger then zzz),  `zzz` seems like a safe choice here.